client/app/cloud/project/compute/infrastructure/cloud-project-compute-infrastructure.config.js

```diff
@@ -0,0 +1,16 @@
+angular.module("managerApp")
+    .config(atInternetControllerDecoratorsProvider => {
+        atInternetControllerDecoratorsProvider.decorate({
+            CloudProjectComputeInfrastructureCtrl: {
+                initInfra (atInternet, controller) {
```

Comment:
I don't see a workaround so I approved anyway but that nesting is killing me.

Reply:
I used to keep consistency with the existing model.
But that part could be review in the futur, and we'll be able to make all changes in one time.